hr_timesheet_employee_analytic_tag/models/account_analytic_line.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        lines = super().create(vals_list)
+        for line in lines:
+            if line.project_id and line.employee_id:
+                employee = line.employee_id.sudo()
+                line.tag_ids += employee.timesheet_analytic_tag_ids
```

Comment:
Why write after create? This is less efficient?

Reply:
Sorry, it has been changed. In the test with Form() an incorrect view was being used and the employee was not defined until after creation.